### PR TITLE
doc: Fix typos in man pages

### PR DESCRIPTION
**Pull request by rountree**
Fixes two trivial documentation typos.

### CODE REVIEW

**Comment by grondo** [comment]
Thanks!!

One last minor thing, the commit message for 81221eb seems to be truncated:

> The prevalent (but not universal)
> capitalization prefers the lower-case v, as does

File: doc/man1/flux-dmesg.adoc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // flux-help-description: manipulate broker log ring buffer
-FLUX-GETATTR(1)
+FLUX-DMESG(1)
 ===============
```
Comment (grondo) [inline doc/man1/flux-dmesg.adoc]: Some asciidoc renderers complain and error out when the number of equal signs below the title don't match the title length (I know, Iknow) (Thanks @garlick for pointing this out)

While asciidoctor doesn't complain, and builds the manpage, it seems to result in a very poorly formatted manpage so this should be fixed:
e.g. right now the page is rendered as:
```
FLUX-DMESG(1)               Flux Command Reference               FLUX-DMESG(1)

FLUX-DMESG(1)

           NAME

               flux-dmesg - access broker ring buffer

               SYNOPSIS
               --------
               *flux* *dmesg* ['OPTIONS']
```
vs
```
FLUX-DMESG(1)               Flux Command Reference               FLUX-DMESG(1)

NAME
       flux-dmesg - access broker ring buffer

SYNOPSIS
       flux dmesg [OPTIONS]


```

Comment (rountree) [reply doc/man1/flux-dmesg.adoc]: @grondo:  Just seeing this now (thanks, @gonsie).  I've fixed the underscoring.  Taking a look at the commit message truncation now.

**Comment by grondo** [comment]
Hm, flux-tree tests are failing in the flux-sched checker. Not sure why
```
FAIL: t2000-tree-basic.t 32 - flux-tree: -T1 in real mode
FAIL: t2000-tree-basic.t 33 - flux-tree: -T1x1 in real mode
FAIL: t2000-tree-basic.t 34 - flux-tree: -T2 with exit code rollup works
PASS: t2000-tree-basic.t 35 - flux-tree: removing qmanager/resource works
ERROR: t2000-tree-basic.t - exited with status 1
```

**Comment by dongahn** [comment]
Hmmm. These three tests are the ones that actually run flux-tree in real mode (not in dry-run mode). Could be `FLUX_RC_EXTRA` related... Anything changed recently for dealing with RCs?

**Comment by grondo** [comment]
> These three tests are the ones that actually run flux-tree in real mode (not in dry-run mode). Could be FLUX_RC_EXTRA related... Anything changed recently for dealing with RCs?

Nothing changed that I know of. But Ah, this PR is out of date, let me force a rebase in case that is it.

**Comment by grondo** [comment]
@mergifyio rebase

**Comment by dongahn** [comment]
There is one change that makes use of exit code setting for `flux job status`. This must be it...

**Comment by grondo** [comment]
@rountree: technically these commits should have `doc:` prefix on the subjects. I could force-push a fix with your permission, though in this case it isn't a big deal.

**Comment by grondo** [comment]
> There is one change that makes use of exit code setting for flux job status. This must be it...

Ah, yeah. Thanks for the hint!

**Pull request merged**